nntrainer/dataset/random_data_producers.cpp

```diff
@@ -79,7 +79,7 @@ const std::string RandomDataOneHotProducer::getType() const {
 unsigned long long
```

Comment:
update datatype to unsigned int or PropNumSamples data type to ull.

Reply:
I'll change this to ull 👍

Reply:
Done